code/__HELPERS/bandetect.dm

```diff
@@ -0,0 +1,36 @@
+#define YOUNG 4
+
+
+/client/proc/join_date_check(var/y,var/m,var/d)
+	var/DBQuery/query = dbcon.NewQuery("SELECT DATEDIFF('[SQLdate()]','[y]-[m]-[d]')")
```

Comment:
If you do
`SELECT DATEDIFF(Now(),'[y]-[m]-[d]')`
It'll internally get a sql-compatible current date when the query is run